frontera/contrib/backends/sqlalchemy/components.py

```diff
@@ -266,7 +266,7 @@ def get_next_requests(self, max_n_requests, partition_id, **kwargs):
         results = []
         for items in six.itervalues(queue):
             for item in items:
-                method = item.method or 'GET'
+                method = item.method or b'GET'
```

Comment:
The same here. method is solely used to initialize Request. And we convert the method argument to bytes in the Request model.